.github/workflows/build-deb-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Test Deb Package
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  pull_request:
+jobs:
+  build-test-deb-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: chmod 755 homeassistant-supervised/DEBIAN/p*
+      - run: dpkg-deb --build --root-owner-group homeassistant-supervised
```

Comment:
Is the `--root-owner-group` really needed here?

Reply:
It seems it is, read step 5 here https://www.internalpointers.com/post/build-binary-deb-package-practical-guide